CHANGELOG.md

```diff
@@ -32,6 +32,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - OIDC Authenticator can now be configured to distribute access tokens with a
   custom time-to-live.
   [cyberark/conjur#2683](https://github.com/cyberark/conjur/pull/2683)
+- List members request (`GET /members`) now produce audit events.
```

Comment:
please move it to the unreleased version

Reply:
done